[0m[[0m[31merror[0m] [0m[0mException in thread "zio-fiber-4" zio.ZIO$Stateful: Stateful(<empty>.Main.run(Main.scala:58),zio.FiberRef$unsafe$$anon$2$$Lambda$6978/0x0000000802257a30@c7ef9eb)[0m
[0m[[0m[31merror[0m] [0m[0m	at <empty>.Main.run(Main.scala:58)[0m
[0m[[0m[31merror[0m] [0m[0m	at <empty>.Main.run(Main.scala:64)[0m
[0m[[0m[31merror[0m] [0m[0m	at <empty>.Main.run(Main.scala:58)[0m
[0m[[0m[31merror[0m] [0m[0m	at <empty>.Main.run(Main.scala:64)[0m
[0m[[0m[31merror[0m] [0m[0m(Compile / [31mrun[0m) Exception in thread "zio-fiber-4" zio.ZIO$Stateful: Stateful(<empty>.Main.run(Main.scala:58),zio.FiberRef$unsafe$$anon$2$$Lambda$6978/0x0000000802257a30@c7ef9eb)[0m
[0m[[0m[31merror[0m] [0m[0m	at <empty>.Main.run(Main.scala:58)[0m
[0m[[0m[31merror[0m] [0m[0m	at <empty>.Main.run(Main.scala:64)[0m
